### PR TITLE
Frame's saved into BE

### PR DESCRIPTION
**Pull request by aliL2000**
This PR makes the image frames properly saved into the BE DB, and renames the reference image model to a convention.

WARNING: DB will need to be dropped and re-done following this change, and further work needs to be done to get image parsing done correctly.

### CODE REVIEW

File: rosen_backend/videoUpload/admin.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
 
 # Register your models here.
-from videoUpload.models import VideoModel,ImageModel
+from videoUpload.models import VideoModel,ReferenceImageModel,ImageFrameModel
 
 admin.site.register(VideoModel)
```
Comment (yuqi88) [inline rosen_backend/videoUpload/admin.py:6]: Question to Ali:
What does the admin do here? Is it for the localhost:8000/admin page?

**Review by yuqi88 — APPROVED** [review]
Looks good to me!

File: rosen_backend/videoUpload/admin.py (hunk 1 above)
Comment (aliL2000) [reply rosen_backend/videoUpload/admin.py:6]: Yes, this registers the models on the admin page so that you can see the models there

**Review by Brenden-Trieu — APPROVED** [review]
LGTM

**Comment by Jeena01** [comment]
lgtm!

**Pull request merged**